sapi/fpm/fpm/fpm_unix.c

```diff
@@ -300,7 +303,67 @@ int fpm_unix_set_socket_permissions(struct fpm_worker_pool_s *wp, const char *pa
 			return -1;
 		}
 		/* Copy the new ACL entry from config */
-		for (i=ACL_FIRST_ENTRY ; acl_get_entry(aclconf, i, &entryconf) ; i=ACL_NEXT_ENTRY) {
+		for (iconf=ACL_FIRST_ENTRY ; acl_get_entry(aclconf, iconf, &entryconf) ; iconf=ACL_NEXT_ENTRY) {
+			if (0 > acl_get_tag_type(entryconf, &tagconf)) {
+				zlog(ZLOG_SYSERROR, "[pool %s] failed to get tag of the ACL entry of the pool", wp->config->name);
+				acl_free(aclfile);
+				return -1;
+			}
+			if (tagconf == ACL_USER) {
+				uidconf = acl_get_qualifier(entryfile);
```

Comment:
The entryfile is not initialized so I assume you wanted to use entryconf here, right?